apps/files/src/components/FilesApp.vue

```diff
@@ -99,8 +70,7 @@ export default {
       fileName: '',
       selected: [],
       breadcrumbs: [],
-      self: {},
-      renameDialogErrorMessage: null
+      self: {}
```

Comment:
Please also remove `self` here. It is unused and should not be part of `data` anyway.